src/main/resources/archetype-resources/src/main/java/adapters/example/ExampleServiceAdapterProxy.java

```diff
@@ -0,0 +1,36 @@
+package ${package}.adapters.example;
+
+import io.knotx.adapter.AbstractAdapterProxy;
+import io.knotx.dataobjects.AdapterRequest;
+import io.knotx.dataobjects.AdapterResponse;
+import io.knotx.dataobjects.ClientResponse;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import rx.Observable;
+
+public class ExampleServiceAdapterProxy extends AbstractAdapterProxy {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(ExampleServiceAdapterProxy.class);
+
+  @Override
+  protected Observable<AdapterResponse> processRequest(AdapterRequest adapterRequest) {
+    final String message = adapterRequest.getParams().getString("message");
+    LOGGER.info("Processing request with message: `{}`", message);
+    /**
```

Comment:
We can add a comment, that actually here all the work of simple Adapter should be done.